### ISSUE TITLE
`VGG_ILSVRC_19_layers.caffemodel` unreachable

### ISSUE THREAD

**Issue by silverkorn** [jcjohnson/neural-style#143]
Is there a mirror for `VGG_ILSVRC_19_layers.caffemodel`?

As for step 3 in the installation process, `sh models/download_models.sh` freezes at "http://www.robots.ox.ac.uk/~vgg/software/very_deep/caffe/VGG_ILSVRC_19_layers.caffemodel"
Same as in my browser...

Thanks! :smiley:

**Comment by hexus** [comment]
This address isn't responding for me at all right now. A mirror would be good.

**Comment by RandomPrecision1** [comment]
Yeah, I just reformatted / reinstalled on my neural net machine, and it seems the entire robots.ox.ac.uk website is down. Hopefully it's just temporary wonkiness?

**Comment by xiaott8888** [comment]
I have the same problem,Is there a mirror for the model?

**Comment by gabormarinov** [comment]
Same for me, I have searched for VGG_ILSVRC_19_layers.caffemodel, but couldn't find any mirror copy.
Please advise.

**Comment by protivinsky** [comment]
The same here. Host is completely unreachable. Hope it will be fixed soon.

**Comment by htoyryla** [comment]
See https://github.com/jcjohnson/neural-style/issues/202

**Comment by gabormarinov** [comment]
Great thanks!
